hack/dataplaneConverter/converter.go

```diff
@@ -0,0 +1,140 @@
+package converter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/hack/dataplaneParser/iptable"
+	"github.com/Azure/azure-container-networking/hack/dataplaneParser/parser"
+)
+
+type Converter struct {
+}
+
+type RuleResponse struct {
+	Chain         string            `json:"chain"`
+	SrcList       []string          `json:"srcList"`
+	DstList       []string          `json:"dstList"`
+	Protocol      string            `json:"protocol"`
+	DPort         string            `json:"dport"`
+	SPort         string            `json:"sport"`
+	Allowed       bool              `json:"allowed"`
+	Direction     string            `json:"direction"`
+	UnsortedIpset map[string]string `json:"unsortedIpset"` //key: ipset name, value: src,dst or dst,dst
+}
+
+type Direction int
+
+func (d Direction) String() string {
+	return [...]string{"EGRESS", "INGRESS"}[d]
```

Comment:
this is a little loosely coupled, any change in this order by other contributors will break all subsequent packages using this direction information.

We might not want to use integer as static but use the string it self as static, that way we can control the string alloc and not worry about the ordering of int to string

Reply:
Also move this to iptables.go